cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
This files is automatically generated by Cypress. We don't need it yet, but I'm committing it in this PR because otherwise it will show up as a new file in git if folks run the tests locally.